claim/views.py

```diff
@@ -36,17 +51,31 @@ def get_claims(request, polygon_id):
 
 @login_required
 def add_claim(request):
+    if not request.POST.get('g-recaptcha-response', False):
```

Comment:
Don't get it. Please create wiki page with description. So far i made:
 - You can disable recaptcha from config
 - If user is logged in - do not ask captcha
 - If user not authorized - ask capthca